xray/utils/resultstable.go

```diff
@@ -152,12 +169,18 @@ func prepareViolations(violations []services.Violation, multipleRoots, isTable,
 // Set multipleRoots to true in case the given vulnerabilities array contains (or may contain) results of several projects or files (like in binary scan).
 // In case multipleRoots is true, the field Component will show the root of each impact path, otherwise it will show the root's child.
 // Set printExtended to true to print fields with 'extended' tag.
-func PrintVulnerabilitiesTable(vulnerabilities []services.Vulnerability, multipleRoots, printExtended bool) error {
+func PrintVulnerabilitiesTable(vulnerabilities []services.Vulnerability, multipleRoots, printExtended, isScan bool) error {
+	log.Output(noContextMessage + "Below are all vulnerabilities detected.")
```

Comment:
I suggest removing this line.

Reply:
I already removed it from dev a week ago together with noContextMessage.

Reply:
Done!